_maps/configs/horizon_apis.json

```diff
@@ -0,0 +1,51 @@
+{
+	"map_name": "Apis-Class Engineering Platform",
```

Comment:
Insert the following:

"$schema": "https://raw.githubusercontent.com/PentestSS13/Pentest/master/_maps/ship_config_schema.json",